src/components/ButtonGroupPicker/index.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import StyledContainer from './styled/container';
+import StyledButtonGroup from './styled/buttonGroup';
+import StyledLegend from './styled/label';
+import { uniqueId } from '../../libs/utils';
+import { Provider } from './context';
+import RenderIf from '../RenderIf';
+import HelpText from './styled/helpText';
+import ErrorText from './styled/errorText';
+
+class ButtonGroupPicker extends Component {
+    constructor(props) {
+        super(props);
+        this.groupNameId = this.props.name || uniqueId('options');
+        this.errorMessageId = uniqueId('error-message');
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getValue() {
+        const { value } = this.props;
+        if (typeof value === 'string') {
+            return [];
+        }
+        return value;
+    }
+
+    getErrorMessageId() {
+        const { error } = this.props;
+        if (error) {
+            return this.errorMessageId;
+        }
+        return undefined;
+    }
+
+    handleOnChange(event) {
+        const { value, checked } = event.target;
+        const { value: values, multiple, onChange } = this.props;
+
+        if (!multiple) {
+            return onChange([value]);
+        }
+
+        if (checked && Array.isArray(values)) {
+            return onChange(values.concat([value]));
+        }
+        if (checked && !Array.isArray(values)) {
+            return onChange([value]);
+        }
+        return onChange(values.filter(valueId => valueId !== value));
+    }
+
+    render() {
+        const {
+            name,
+            className,
+            style,
+            label,
+            multiple,
+            children,
+            size,
+            error,
+            bottomHelpText,
+        } = this.props;
+
+        const context = {
+            onChange: this.handleOnChange,
+            values: this.getValue(),
+            type: multiple ? 'checkbox' : 'radio',
+            name: name || uniqueId('option'),
+            ariaDescribedBy: this.getErrorMessageId(),
+        };
+
+        return (
+            <StyledContainer className={className} style={style}>
+                <RenderIf isTrue={!!label}>
+                    <StyledLegend>{label}</StyledLegend>
+                </RenderIf>
+                <StyledButtonGroup size={size} role="group">
+                    <Provider value={{ ...context }}>{children}</Provider>
+                </StyledButtonGroup>
+                <RenderIf isTrue={!!bottomHelpText}>
+                    <HelpText alignSelf="center">{bottomHelpText}</HelpText>
+                </RenderIf>
+                <RenderIf isTrue={!!error}>
+                    <ErrorText id={this.getErrorMessageId()}>{error}</ErrorText>
+                </RenderIf>
+            </StyledContainer>
+        );
+    }
+}
+
+ButtonGroupPicker.propTypes = {
+    className: PropTypes.string,
```

Comment:
We need to add the descriptions of the props :)
Let's use the descriptions defined on the API doc.